doc/Messaging.md

```diff
@@ -0,0 +1,148 @@
+# Vitess Messaging
+
+# Overview
+
+Vitess messaging gives the application an easy way to schedule and manage work that needs to be performed asynchronously. Under the covers, messages are stored in a traditional MySQL table and therefore enjoy the following properties:
```

Comment:
Can you format this file with < 80 characters per line? My emacs does that for me...